cms_agents/reduction_agent.py

```diff
@@ -64,26 +65,36 @@ def publish_reduced_documents(reduced, metadata, reduced_publisher):
     reduced_publisher("stop", cr.compose_stop())
 
 
-def respond_to_stop_with_reduced():
+def respond_to_stop_with_reduced(consumer_topic, testing):
 
     kafka_config = nslsii.kafka_utils._read_bluesky_kafka_config_file(config_file_path="/etc/bluesky/kafka.yml")
 
     cms_tiled_client = from_profile("cms")
-    cms_sandbox_tiled_client = from_profile('cms_bluesky_sandbox')
-    reduced_publisher = Publisher(
-        key='',
-        topic="cms.bluesky.reduced.documents",
-        bootstrap_servers=",".join(kafka_config["bootstrap_servers"]),
-        producer_config=kafka_config["runengine_producer_config"],
-    )
-
-    def do_both(name, doc):
-        cms_sandbox_tiled_client.v1.insert(name, doc)
-        reduced_publisher(name, doc)
-
+    
+    if testing:
+        def do_both(name, doc):
```

Comment:
Can we make `do_both` descriptive? Something like `publishing_method`?

Reply:
done